impeller/entity/shaders/radial_gradient_fill.frag

```diff
@@ -3,21 +3,24 @@
 // found in the LICENSE file.
 
 #include <impeller/texture.glsl>
+#include <impeller/transform.glsl>
 
 uniform GradientInfo {
   vec2 center;
   float radius;
   vec4 center_color;
   vec4 edge_color;
   float tile_mode;
+  mat4 matrix;
 } gradient_info;
 
 in vec2 interpolated_vertices;
 
 out vec4 frag_color;
 
 void main() {
-  float len = length(interpolated_vertices - gradient_info.center);
+  vec2 transformed_vertices = transform(gradient_info.matrix, interpolated_vertices);
```

Comment:
Here and everywhere else, this transform could just be done in the vertex shader before throwing it over to the vertex shader.

Reply:
Also, it seems the gradient vertex shaders have turned out to be identical. If they continue being identical, consider deleting radial_gradient_fill and sweep_gradient_fill and just use the gradient_fill vertex shader for all of the pipelines.

Reply:
Done.